form_utils/forms.py

```diff
@@ -13,8 +13,9 @@
     from django.forms.utils import flatatt, ErrorDict
 except ImportError: # Django < 1.9 compatibility
     from django.forms.util import flatatt, ErrorDict
-from django.utils import six
+import six
```

Comment:
I know you didn't write these commits but would be nice to move this up to group with the other non-django imports

Reply:
I'll group these imports in a separate PR because this PR doesn't actually create a branch to which I can commit. I would have to merge the changes by CLI, not in the web interface.